# Message describing the full Topological Navigation Map

Header header 

TopoNavNodeMsg[] nodes
TopoNavEdgeMsg[] edges
